build-tools/packages/version-tools/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+	test: {
+		reporters: [["junit", { suiteName: "version-tools" }], "default"],
+		outputFile: {
+			junit: "nyc/junit-report.xml",
```

Comment:
Nit: might be a time to reconsider the output directory naming for test output. I liked `playwright`'s default of `test-results`. We could always do `test-results/code-coverage` or something if we'd prefer to separate the coverage reports from other reports.

Reply:
Good point. One challenge is that the "nyc" folder is referred to in a lot of places, including in CI workflows. So that will be a relatively large undertaking.

Reply:
I feel similarly about the dist/lib folders we use for build output. @ChumpChief and I have discussed changing that many times in the past, but it has the same "we have to edit the world to make it happen" problem. :(